export/tasks_generator_test.go

```diff
@@ -327,59 +335,64 @@ func TestTasksGenerator_Simple(t *testing.T) {
 	}
 }
 
+func getGeneratedID(path string) string {
```

Comment:
This copies code from the system under test.  Consider just exporting it -- maybe inside a `ForTesting` object.  Or strip the generated "random" values before comparing.

Reply:
👍